src/inbox/mod_inbox.erl

```diff
@@ -56,6 +61,14 @@
                       Username :: jid:luser(),
                       Server :: jid:lserver().
 
+-type get_inbox_params() :: #{
+        start => erlang:timestamp(),
+        'end' => erlang:timestamp(),
+        order => asc | desc
+       }.
+
+-export_type([get_inbox_params/0]).
```

Comment:
👍 for exporting the type. I really don't like types from `.hrl` files. I know they were not introduced in this PR.